packages/inula-adapter/examples/$parent/list.jsx

```
@@ -0,0 +1,27 @@
+import TableItem from 'table-item';
+import { expose } from 'vue-inula';
+
+export default function (props) {
+  const { items } = props;
+
+  const filteredIdems = items;
+
+  function setFilterType(type) {
+    filteredItems = items.filter(item => item.type === type);
```

ESLint

[error] 'filteredItems' is not defined.

(no-undef)